cmd/kueue/main.go

```diff
@@ -370,6 +372,14 @@ func apply(configFile string) (ctrl.Options, configapi.Configuration, error) {
 				errorlist = append(errorlist, field.NotSupported(path, framework, availableFrameworks))
 			}
 		}
+
+		path = field.NewPath("integrations", "externalFrameworks")
+		for idx, name := range cfg.Integrations.ExternalFrameworks {
```

Comment:
Let's also have some validation for not repeating the same type in both integration list and externalFrameworks.